<launch>    

    <!-- Configuration of Jackal which you would like to simulate. -->
    <arg name="ns" default="" />    

    <!-- UWB simulator node -->
    <node pkg="jackal_range" name="UWB_sim" type="UWB_sim.py" respawn="false" respawn_delay="2">
        <param name="anchors_topic_name" type="string" value="AnchorsInfoReal" />
        <param name="odometry_topic_name" type="string" value="ground_truth/state" />
        <param name="range_meas_cov" type="double" value="0.20"/> 
    </node>    

</launch>